### ISSUE TITLE
StringUtil.normalize is stripping blank lines from commands

### ISSUE THREAD

**Issue by kshakir** [broadinstitute/wdl4s#230]
Empty lines in command blocks are being removed. While the spec says that leading whitespace is [removed](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#stripping-leading-whitespace), it does not specify that blank lines should be removed. Thus the `StringUtil.normalize` [regex](https://github.com/broadinstitute/wdl4s/blob/850d0f1304373fda8c3cb44b16620969a5f2260b/wom/src/main/scala/wdl4s/wdl/util/StringUtil.scala#L35) should use [`\r?\n`](http://www.mkyong.com/java/java-how-to-split-string-by-new-line/) instead of `[\r\n]+`.

As a workaround, prepending a space on the otherwise blank lines will keep `StringUtil` from removing the line.

```
# no spaces on line in between gets collapsed, results in "hello echo world"
command {
  echo hello \

  echo world
}
```

```
# two spaces on line in between do not get collapsed, results in "hello" and then "world"
command {
  echo hello \
  
  echo world
}
```

**Comment by yfarjoun** [comment]
Cool work-around! Thanks!

**Comment by yfarjoun** [comment]
was this fixed ??

**Comment by kshakir** [comment]
Yes, it is fixed in the next version of cromwell by #231.
